scripts/Password_Manager/main.py

```diff
@@ -0,0 +1,106 @@
+import random
+import base64
+import pickle
+from Crypto.Cipher import AES
+import hashlib
+
+
+def AutoGen(number):
+    passwd = ""
+    for i in range(number):
+        passwd += chr(random.randint(33, 126))
+    return passwd
+
+
+def encrypt_file(message):
+
+    password = "password".encode()
```

Comment:
Can we add a master password sort of feature for encryption and decryption instead of hard coding these stuff in the code itself?

Reply:
user can change the code if they want

Reply:
People can just check the source code and figure out the hardcoded key, mode and IV. While it is fine for mode and IV, I think we can play around with the key and turn it into a master password.

Reply:
idk how to implement that

Reply:
```py
from PBKDF2 import PBKDF2
from Crypto.Cipher import AES
import os
salt = os.urandom(8)    # 64-bit salt
key = PBKDF2("This passphrase is a secret.", salt).read(32) # 256-bit key
iv = os.urandom(16)     # 128-bit IV
cipher = AES.new(key, AES.MODE_CBC, iv)
```
Try taking hints from this snippet from stackoverflow

Reply:
PBKDF2 allows you to follow this model master pass -> encryption key -> encrypting data using the key and vice versa

Reply:
I think this works as a demo

Reply:
As they say "A software never gets finished"